docs/simbad/simbad.rst

```diff
@@ -602,6 +602,62 @@ And the columns in the output can be reset to their default value with
     A detailed description on the ways to add fluxes is available in the
     :ref:`optical filters` section.
 
+Measurement tables
+^^^^^^^^^^^^^^^^^^
+
+Some field names start with ``mes``. These denote what SIMBAD calls a
+"measurement table". These tables store the history on past measurements of a physical
+parameter for each object.
+
+Let's look at the star ``HD 200207`` with the parallax measurements table ``mesplx``:
+
+.. doctest-remote-data::
+
+    >>> from astroquery.simbad import Simbad
+    >>> simbad = Simbad()
+    >>> simbad.add_votable_fields("mesplx")
+    >>> hd_200207 = simbad.query_object("HD 200207")
+    >>> hd_200207[["main_id", "mesplx.plx", "mesplx.plx_err", "mesplx.bibcode"]]
+    <Table length=5>
+     main_id  mesplx.plx mesplx.plx_err    mesplx.bibcode
+                 mas          mas
+      object   float32      float32            object
+    --------- ---------- -------------- -------------------
+    HD 200207     3.4084         0.0195 2020yCat.1350....0G
+    HD 200207     3.4552         0.0426 2018yCat.1345....0G
+    HD 200207       3.35           0.76 1997A&A...323L..49P
+    HD 200207       3.72           0.62 2007A&A...474..653V
+    HD 200207       3.25           0.22 2016A&A...595A...2G
+
+This field adds one line per parallax measurement: five articles have measured it
+for this star.
+
+If you are only interested in the most precise measure recorded by the SIMBAD team, it
+is selected and most of the time available via an other field (not starting by ``mes``)
+that will only give one line per object with the most precise currently known value.
+The exceptions for which there is no pre-selection of a best value are metallicity
+measurements ``mesfe_h`` and stellar variability reports ``mesvar``.
+
+Here, we could have done:
+
+.. doctest-remote-data::
+
+    >>> from astroquery.simbad import Simbad
+    >>> simbad = Simbad()
+    >>> simbad.add_votable_fields("parallax")
```

Comment:
is there a way to find the list of paired `mes`/non-`mes` quantities?  The link between `mesplx` and `parallax` isn't obvious.

Reply:
There is not, especially for the velocities/distance/rotation ones which translate from three distinct measurement tables into one big field rvz
What about adding a table here? It could become obsolete with the evolution of the database... But could be clearer for the time being?

Reply:
That would be great.  Perhaps long-term, if you guys are planning to keep making more of these, you'll need a user-accessible table anyway? 

But in any case, it's your choice what to do here; this shouldn't hold up the PR

Reply:
Actually, found out by doing the table and asking my colleague that most of the measurement tables don't have a selected version (one less than half). So I changed the text and added the table. The names are quite chaotic, so the table is a welcome addition, thanks for raising the issue!